docker-compose.yml

```diff
@@ -27,3 +27,6 @@ services:
       - "./db:/opt"
     ports:
       - "5432:5432"
+    environment:
+      - POSTGRESS_USER=sidewalk
+      - POSTGRES_PASSWORD=sidewalk
```

Comment:
Can you fix this file to end with a newline character, as per [style guide](https://github.com/ProjectSidewalk/SidewalkWebpage/wiki/Style-Guide#general-guidelines)?